lit_gpt/config.py

```diff
@@ -1037,4 +1037,31 @@ def norm_class(self) -> Type:
         configs.append(copy)
 
 
+###################
+# StatNLP Research
+###################
+
+tiny_LLaMA = [
```

Comment:
the original source is TinyLlama, so to be consistent this should be

```suggestion
tinyllama = [
```